### DISCUSSION TITLE
libertube won't work at this moment

### DISCUSSION THREAD

**Discussion by ko-byte** [libre-tube/LibreTube#6357] [Q&A]
i can't use my libertube anymore, every video I clicked on it shows up a message. 
i tried all of my best my best to solve a problem but it's still not working and I'm trying to uninstall my application and install it again and again I face the problem, tells me I have to sign in to make sure that I'm not about but I swear my god I'm not about but I didn't understand why?  without liber tube my live went like a mess, cuz i hate youtube and also snaptube or pipetube. 
so i hope you solve that problem and i will be appreciated you for helping me 🙏

**Comment by ko-byte** [comment]
![Screenshot_2024-08-14-21-16-39-13_54f92cd6866f5233ab0db4392b806878](https://github.com/user-attachments/assets/652e862b-5123-4ba3-a2c7-f9ad3f405168)

**Reply by MrPusemukel** [reply]
Hi , here the same Problem.

**Comment by Raj2032** [comment]
Try his suggestion:
https://github.com/libre-tube/LibreTube/issues/6389

**Comment by blublu80** [comment]
You need to register on Settings/ Istance/Account

anything to register a username and a password possibly long and complex ( but remember it)

Then on proxy enable “Disable Piped proxy” then enable "Local stream extraction" ... That is all now all the videos go to me , I hope it is the same for you too
